Saule/ConfigureFormattersEnum.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Saule
+{
+    /// <summary>
+    /// Configuration enum for formatters
```

Comment:
This description isn't very clear. Maybe describe the purpose rather than what it does?